### PR TITLE
Updates to getting started page inc new images

### LINKED ISSUE #135: Getting started: Add a description and diagram of the OCDS model of the contracting process
The Getting started section does a good job of introducing the structure of the CoST IDS, before describing the structure of OC4IDS.

It would be good if it also briefly described the structure of the OCDS schema to help users interpret the 'How is OC4IDS structured?' diagram.

I suggest adding the following text and image at the end of the [How does contracting data fit in?](http://standard.open-contracting.org/infrastructure/latest/en/projects/#how-does-contracting-data-fit-in) section:

> In the Open Contracting Data Standard a contracting process has 5 stages:
>
> image
>
> These stages all fit within the procurement and implementation stages of the CoST IDS.

### PR DESCRIPTION
**Pull request by odscrachel**
Updates to getting started page based on [draft document](https://docs.google.com/document/d/12OVextU1PJBRfnF_yYW9XpoewilvLbJXTn9GpjJRG2s/edit#heading=h.xma39eesi716) 

Closes #135 

Related issue: CRM-5964

### CODE REVIEW

**Comment by jpmckinney** [comment]
@odscrachel The PR is passing now :)

**Comment by jpmckinney** [comment]
Preview: https://standard.open-contracting.org/staging/infrastructure/getting_started_update/en/projects/

**Comment by jpmckinney** [comment]
LGTM but I prefer that @duncandewhurst reviews.

File: docs/projects/index.md [hunk 1]
```diff
@@ -1,123 +1,150 @@
...
     .. markdown::
 
-      The term "project" is used in many contexts to mean different things, for example:
-
-      * a larger programme of work, like the construction of an entire highway
-      * an investment project, like in National Public Investment Systems (SNIP) in Latin America
-      * a project code in a government's budget
-
-      However, to correctly implement and interpret OC4IDS, the above definition of an infrastructure project needs to be adhered to.
+      The term “project” is used in many contexts to mean different things. In OC4IDS, the term ‘project’ only refers to an ‘infrastructure project’ and should not be applied to an investment project, investment program, or budget code.
```
Comment (duncandewhurst) [inline docs/projects/index.md]: Correct inconsistent use of double/single quotes from Google Doc:

```suggestion
      The term "project" is used in many contexts to mean different things. In OC4IDS, the term "project" only refers to an infrastructure project and should not be applied to an investment project, investment program, or budget code.
```

**Review by duncandewhurst — APPROVED** [review]
I'm happy for this to be merged once the suggestion is committed. Thanks @odscrachel!

**Comment by duncandewhurst** [comment]
@jpmckinney did you also want to review before merging?

**Comment by jpmckinney** [comment]
I did a quick review above. All good!

**Pull request merged**